src/js/views/globals/search/patient-search_views.js

```diff
@@ -99,16 +99,24 @@ const PicklistItem = View.extend({
         {{status}}
       </div>
       <div class="patient-search__picklist-item-result-value u-text--overflow">
-        {{matchText match.value search}}
+        {{#if (formatMessage (intlGet "globals.search.patientSearchViews.itemView.showMatchData") label=matchLabelNormalized)}}
+          {{matchText match.value search}}
+        {{/if}}
       </div>
       <div class="patient-search__picklist-item-result-label u-text--overflow">
-        {{match.label}}
+        {{#if (formatMessage (intlGet "globals.search.patientSearchViews.itemView.showMatchData") label=matchLabelNormalized)}}
+          {{match.label}}
+        {{/if}}
       </div>
     </div>
   `,
   templateContext() {
+    const match = this.model.get('match');
+
     return {
+      name: `${ this.model.get('first_name') } ${ this.model.get('last_name') }`,
       search: this.state.get('search'),
+      matchLabelNormalized: match.label.replace(/ /g, '_'),
```

Comment:
I think the simpler way to accomplish this will be
```
shouldShowMatch() {
  const label = this.model.get('label');
  const i18n = intl.globals.search.patientSearchViews;
  return label !== i18n.labelName && label !== i18n.labelDob;
}
```
and just `{{#if shouldShowMatch}}`

Reply:
That works well, updated to that 👍